examples/file-upload/index.ts

```diff
@@ -1,7 +1,7 @@
 import { GraphQLServer } from 'graphql-yoga'
 import { createWriteStream } from 'fs'
 import * as mkdirp from 'mkdirp'
-import shortid from 'shortid'
+import * as shortid from 'shortid'
```

Comment:
Since we just use the generate function inside `shortid` why don't you use `import { generate } from 'shortid'` and change from `shortid.generate()` to `generate()`.